operator/deploy/operator.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: v1
```

Comment:
Please add a TODO here about moving this to `deploy` and enabling kustomize.

Reply:
Added TODOs about kustomization, and location of this yaml file.